test/ssl-tests/06-sni-ticket.conf.in

```diff
@@ -0,0 +1,74 @@
+# -*- mode: perl; -*-
+# Copyright 2016-2016 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+## Test version negotiation
+
+package ssltests;
+
```

Comment:
Please add

```
use strict;
use warnings;
```

I think that I may have forgotten that myself in some of the confs but I'm trying to remember now :)